reference/7.1/Microsoft.PowerShell.Management/Add-Content.md

```diff
@@ -233,6 +233,10 @@ pages (like `-Encoding 1251`) or string names of registered code pages (like
 `-Encoding "windows-1251"`). For more information, see the .NET documentation for
 [Encoding.CodePage](/dotnet/api/system.text.encoding.codepage?view=netcore-2.2).
 
+> [!NOTE]
+> **UTF-7*** is no longer recommended to use and in PowerShell 7.1, a warning will be written if you
+> specify `utf7` for the **Encoding** parameter.
```

Comment:
Break this into two sentences.   "UTF-7 is no longer recommended to use.  In PowerShell 7.1, a warning is written..." (this also removes the passive.

Reply:
Fixed

Reply:
Still one long sentence.

Reply:
lol it doesn't show the updates here (Not a fan of how that works.) You have to go view the actual content. Should be updated?

Reply:
You changed some of them with a period, but not all of them.  And it reads funny to me.  Here's what I think it should be:

```
> **UTF-7*** is no longer recommended to use. In PowerShell 7.1, a warning is written if you
> specify `utf7` for the **Encoding** parameter.
```